services/stackexchange/stackexchange-monthlyquestions.tester.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const Joi = require('joi')
+const withRegex = re => Joi.string().regex(re)
+const isMonthMetric = withRegex(/^[1-9][0-9]*[kMGTPEZY]?\/(month)?$/)
```

Comment:
To avoid the need for another regex, you could use `isMetricOverTimePeriod` even though it's slightly more lenient.